examples/kubernetes-hello-world/kubernetes_hello_world.md

```diff
@@ -11,10 +11,7 @@ For all the supported key value properties, see [Code to Cloud Specification](ht
 ::: code Cloud.toml :::
 
 Execute the `bal build` command to build the Ballerina package. Code to Cloud generates only one container per package.
->**Note:** The macOS users with Apple silicon chip need set an environment variable `DOCKER_DEFAULT_PLATFORM` to `linux/amd64`, before building the image. This is because the Ballerina Docker image is not supported on Apple silicon chips yet.
-> ```
-> export DOCKER_DEFAULT_PLATFORM=linux/amd64
-> ```
+>**Note:** macOS users with Apple Silicon chips need to set an environment variable named `DOCKER_DEFAULT_PLATFORM` to `linux/amd64`, before building the image. This is because the Ballerina Docker image is not supported on Apple Silicon chips yet.Run `export DOCKER_DEFAULT_PLATFORM=linux/amd64` to set the environment variable.
```

Comment:
```suggestion
>**Note:** macOS users with Apple Silicon chips need to set an environment variable named `DOCKER_DEFAULT_PLATFORM` to `linux/amd64`, before building the image. This is because the Ballerina Docker image is not supported on Apple Silicon chips yet. Run `export DOCKER_DEFAULT_PLATFORM=linux/amd64` to set the environment variable.
```